src/containers/registration.jsx

```diff
@@ -1,6 +1,6 @@
 // @flow
 import React, { Component } from "react";
-import InputLabel from "../components/inputLabel";
+import InputLabel from "../components/input";
```

Comment:
Seems like it is not 'InputLabel' it is 'Input'. So, name this imported component property